src/plugins.ts

```diff
@@ -235,8 +234,7 @@ export async function runPluginsOnBatch(server: PluginsServer, batch: PluginEven
                     errored = true
                     await processError(server, pluginConfig, error, returnedEvents[0])
                 }
-                const ms = Math.round((performance.now() - startTime) * 1000) / 1000
-                logTime(pluginConfig.plugin?.name || 'noname', ms, errored)
+                logTime(pluginConfig.plugin?.name || 'noname', startTime - performance.now(), errored)
```

Comment:
Aren't we subtracting a larger (newer) value from a smaller (older) one?

Reply:
You're right! I See you're in the middle of fixing things, so feel free to reverse this!